.prettierignore

```diff
@@ -1,2 +1,3 @@
 README.md
+doc
```

Comment:
Why is `doc` ignored?

Reply:
That was before [@metarhia/doc](https://github.com/metarhia/doc) supported prettier style. Now it's fixed.